package.json

```diff
@@ -20,7 +23,8 @@
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:e2e": "jest --config ./test/jest-e2e.json",
     "start:cli": "ts-node src/cli/index.ts",
-    "dev:cli": "nodemon --quiet --watch 'src/**/*.ts' --exec 'ts-node' src/cli/index.ts"
+    "dev:cli": "nodemon --quiet --watch 'src/**/*.ts' --exec 'ts-node' src/cli/index.ts",
+    "deploy": "npm ci && npm run build && npm run start:prod"
```

Comment:
Just a small question, is there a good reason to use npm here and not yarn? I read it's a good practice not to mix up package managers 🙂

Reply:
Yarn isn't installed by default on the aws EB/node.js so i choose the faster solution ! But seems the good way to go to use only yarn. We could create an issue related to this and merge this branch now ?